medicines/search-client/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "search-client"
+version = "0.1.0"
+authors = ["Matt Doughty <matt.doughty@mhra.gov.uk>", "Craig Anderson <craig.anderson@red-badger.com>"]
```

Comment:
is this intentional?

Reply:
Is what intentional?

Reply:
having specific developer names

Reply:
I thought we had those in all of our Cargo.toml files?

Reply:
You need an email address in your `Cargo.toml` in order to publish to crates.io.